README.md

```diff
@@ -8,47 +8,46 @@ you discover that you've got "***web super powers***"!!
-The problem is that
+Instead _most_ people do (_manual_) "click though" or "visual regression" testing.
+It turns out us [humans can literally be blind](https://en.wikipedia.org/wiki/Change_blindness) to spotting changes during manual testing.
 
+Other uses? [Automated testing can help with web scraping too...](https://hackernoon.com/nightmarishly-good-scraping-with-nightmare-js-and-async-await-b7b20a38438f)
 
 ## _What_?
 
 This guide is _an introduction to_ "***web super powers***"
 _for **complete beginners**_.<br />
 _Unusually_ for a [dwyl "learn" tutorial](https://github.com/search?q=org%3Adwyl+learn)
-will be focussing on learning **_two_ tools**
+we will be focussing on learning **_two_ tools**
 in the next **20-30 minutes**. <br />
-(_we usually try and learn/teach **one** thing at a time,
-  but in this case the tools go together like electricity and a **lightbulb**_!)
+(_We usually try and learn/teach **one** thing at a time,
+  but in this case the tools go together like electricity :zap: and a **lightbulb**_:bulb:!)
 
-### Daydream
+  ### Daydream
 
-Daydream is _extension_ for the Google Chrome web browser that allows
-you to record your actions into a script you can re-play!
-
-+ Google Chrome Extension store: https://github.com/segmentio/daydream#google-chrome
-+ GitHub: https://github.com/segmentio/daydream
+  Daydream is an _extension_ for the Google Chrome web browser that allows
+  you to record your actions into a script you can re-play!
 
+  + Google Chrome Extension store: https://github.com/segmentio/daydream#google-chrome
+  + GitHub: https://github.com/segmentio/daydream
 
 ### Nightmare.js
 
-<!-- ![nightmare logo](https://cloud.githubusercontent.com/assets/194400/23049531/a0c1372a-f4b4-11e6-96a6-a38637f245a4.png) -->
-
-Nightmare.js is a JavaScript library that
-lets you automate browser interaction! <br />
-Anything that a _human_ can do in a web browser can be automated by writing
-a small Nightmare.js script.
+  <!-- ![nightmare logo](https://cloud.githubusercontent.com/assets/194400/23049531/a0c1372a-f4b4-11e6-96a6-a38637f245a4.png) -->
 
-+ Website: http://nightmarejs.org (_which **really** does not do it justice!_)
-+ GitHub: https://github.com/segmentio/nightmare
+  Nightmare.js is a JavaScript library that
+  lets you automate browser interaction! <br />
+  Anything that a _human_ can do in a web browser can be automated by writing
+  a small Nightmare.js script.
 
+  + Website: http://nightmarejs.org (_which **really** does not do it justice!_)
+  + GitHub: https://github.com/segmentio/nightmare
 
 #### Not Covered
 
-_Awkwardly_, Google _recently_ decided
-to call their Virtual Reality (VR) project "_Daydream_" ...
-see: [google.com/**daydream**](https://vr.google.com/daydream) <br />
+_Awkwardly_, Google recently decided
+to call their Virtual Reality (VR project "_Daydream_" ...
+see [google.com/**daydream**](https://vr.ogle.com/daydream),
```

Comment:
https://vr.ogle.com/daydream times out ... 
![image](https://cloud.githubusercontent.com/assets/194400/24529454/b382bc98-15a3-11e7-9e5b-cd29464ccc36.png)
is this a typo?